apps/financial-aid/web-veita/src/components/NewUserModal/NewUserModal.tsx

```diff
@@ -83,7 +85,13 @@ const NewUserModal = ({
         onStaffCreated()
       })
     } catch (e) {
-      setState({ ...state, hasSubmitError: true })
+      setState({
+        ...state,
+        hasSubmitError: true,
+        hasNationalIdError:
+          (e as ApolloError).graphQLErrors[0]?.extensions?.response.status ===
+          400,
```

Comment:
I know we do it like this in other places, but is there another way of getting the error?

Reply:
This is is only place in the error object that has the status as a number